src/test/java/com/epam/training/student_anton_lapushenko/WebDriver/Task2/CreatePasteTest.java

```diff
@@ -1,32 +1,44 @@
 package com.epam.training.student_anton_lapushenko.WebDriver.Task2;
 
+import com.epam.training.student_anton_lapushenko.WebDriver.Task2.enums.ExpirationTime;
+import com.epam.training.student_anton_lapushenko.WebDriver.Task2.pages.CreateNewPastePage;
+import com.epam.training.student_anton_lapushenko.WebDriver.Task2.pages.PastePage;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class CreatePasteTest extends BaseChromeTest {
-
+public class CreatePasteTest extends BaseTest{
     @BeforeEach
     public void navigate() {
         driver.get("https://pastebin.com/");
     }
 
     @Test
     public void testCreatePaste() {
-        CreateNewPastePage createNewPastePage = new CreateNewPastePage(driver);
-
         String text = "git config --global user.name \"New Sheriff in Town\"" +
                 "\ngit reset $(git commit-tree HEAD^{tree} -m \"Legacy code\")" +
                 "\ngit push origin master --force";
 
         String nameOrTitle = "how to gain dominance among developers";
 
-        PastePage pastePage = createNewPastePage.createPasteWithBashHighlighting(text, nameOrTitle, ExpirationTime.TenMin);
+        PastePage pastePage = createPasteWithBashHighlighting(text, nameOrTitle, ExpirationTime.TenMin);
 
         assertEquals(nameOrTitle, pastePage.getNameOrTitle());
         assertEquals(ExpirationTime.TenMin.text, pastePage.getExpireTime());
         assertEquals("Bash", pastePage.getSyntaxHighlighting());
         assertEquals(text, pastePage.getText());
     }
+
+    public PastePage createPasteWithBashHighlighting(String text, String nameOrTitle, ExpirationTime expirationTime) {
+        CreateNewPastePage createNewPastePage = new CreateNewPastePage(driver);
+        createNewPastePage.setText(text);
+        createNewPastePage.setBashHighlighting();
+        createNewPastePage.setExpirationTime(expirationTime);
+        createNewPastePage.setNameOrTitle(nameOrTitle);
+        createNewPastePage.clickSubmit();
+        return new PastePage(driver);
+    }
```

Comment:
This is not the test code. As I mentioned in the previous PR this method can be located in the Business Object class. But for this task additional layer not required. 
You can read more about the business layer here - https://www.tutorialspoint.com/what-is-the-business-logic-layer
You can create a new one, for example, CreatePasteSteps, and move it to this class.
This class can be located in src/main/steps package